core/lib/multivm/src/versions/era_vm/vm.rs

```diff
@@ -633,8 +646,9 @@ impl<S: ReadStorage> era_vm::store::ContractStorage for World<S> {
                 .clone(),
         ))
     }
-    fn hash_map(&self) -> Result<HashMap<U256, Vec<U256>>, StorageError> {
-        Ok(self.contract_storage.as_ref().clone().into_inner())
+
+    fn decommited_hashes(&self) -> Result<HashSet<U256>, StorageError> {
+        Ok(self.decommited_hashes.clone())
     }
```

Comment:
There's no need to wrap this function inside of a `Result`:
```suggestion
    fn decommited_hashes(&self) -> HashSet<U256> {
        self.decommited_hashes.clone()
    }
```